packages/frontend/app/dashboard/components/admin-dashboard.tsx

```diff
@@ -11,45 +11,41 @@ import {
   placeholderInformationBoard,
   placeholderDuties,
 } from "../../../lib/sample-data";
-import { DashboardItems } from "../../../lib/types/dashboard-item";
 
 import AnimateCard from "./animate-card";
-import CongEventsCard from "./dashboard-components/cong-events-card";
-import InformationCard from "./dashboard-components/information-card";
-import MeetingDutiesCard from "./dashboard-components/meeting-duties-card";
-import PublicWitnessingCard from "./dashboard-components/public-witnessing-schedule-card";
-import UserInfoCard from "./dashboard-components/user-info-card";
+import CongEventsCard from "./sections/cong-events-card";
+import InformationCard from "./sections/information-card";
+import MeetingDutiesCard from "./sections/meeting-duties-card";
+import PublicWitnessingCard from "./sections/public-witnessing-schedule-card";
+import UserInfoCard from "./sections/user-info-card";
+import { DashboardComponentProps, DashboardItem } from "./types";
 
 import { Button } from "@/components/ui/button";
 import {
   Card,
   CardContent,
   CardDescription,
-  CardFooter,
   CardHeader,
   CardTitle,
 } from "@/components/ui/card";
-import { User_WithCongregation } from "@/lib/types/compositions";
 
-function AdminDashboard({
-  currentUser,
-}: {
-  currentUser: User_WithCongregation;
-}) {
+export function AdminDashboard({ currentUser }: DashboardComponentProps) {
   const router = useRouter();
 
-  const userDashboardItems: DashboardItems = [
-    <UserInfoCard
-      userInfo={{
-        name: "John Doe",
-        email: "example@gmail.com",
-        congregation: "London, Uxbridge",
-      }}
-    />,
-    <MeetingDutiesCard data={placeholderDuties} />,
-    <InformationCard data={placeholderInformationBoard} />,
-    <CongEventsCard data={placeholderCongEvents} />,
-    <PublicWitnessingCard />,
+  const userDashboardItems: DashboardItem[] = [
+    () => (
```

Comment:
What's the difference here?

Reply:
This is called lazy evaluation. I didn't want the components to be created all at once on every render, instead just functions to create them. This allows them to be created dynamically. Probably a negligible effect but just didn't want to use up too much memory

Reply:
Especially since components are quite heavy in general already